packages/common/src/index.ts

```diff
@@ -1,3 +1,3 @@
 // import './test';
 
-export * from './Registry';
+export * from './registry';
```

Comment:
it is not related to this PR but idk - without that change I am unable to build project at all

Reply:
@michaldziuba03 thanks for noticing, @jamesL92 just fixed it at #473, so we go ahead to merge this cool new formatter :)